core/src/context.rs

```diff
@@ -91,7 +91,7 @@ impl Context {
 
     #[cfg(feature = "parallel")]
     fn reset_stack(&self) {
-        unsafe { qjs::JS_ResetStackPointer(self.ctx) };
+        unsafe { qjs::JS_UpdateStackTop(qjs::JS_GetRuntime(self.ctx)) };
```

Comment:
Somewhat pedantic note.
Retrieving the runtime pointer this way is unsafe since if the function is called before the runtime is locked it could result in a race condition.
Not that this function is used ever before acquiring a lock, but it might be somewhat more ideologically pure to have this function take the locked runtime as an argument.

Reply:
Ohh, seems I planned move thi function to `Inner` of `Runtime` but forgot to do it :)